spec/models/articles_spec.rb

```diff
@@ -0,0 +1,114 @@
+require 'rails_helper'
+
+RSpec.describe Article, type: :model do
+  describe 'validations' do
+    before { subject.valid? }
```

Comment:
This line is extra. we already execute invalid? method with predicate matcher `is_expected.to be_invalid`